src/Compilers/Core/Portable/Operations/BasicBlock.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis.PooledObjects;
+
+namespace Microsoft.CodeAnalysis.Operations
+{
+    /// <summary>
+    /// PROTOTYPE(dataflow): Add documentation
+    /// </summary>
+    public enum BasicBlockKind
+    {
+        Entry,
+        Exit,
+        Block
+    }
+
+    /// <summary>
+    /// PROTOTYPE(dataflow): Add documentation
+    /// </summary>
+    public sealed class BasicBlock
```

Comment:
Nit: are we planning other kinds of blocks? If not consider just calling "Block".

Reply:
Basic block is a well known term.

---
In reply to: [162137677](https://github.com/dotnet/roslyn/pull/24263#discussion_r162137677) [](ancestors = 162137677)